admin/customers.php

```diff
@@ -1574,6 +1574,8 @@ class="btn btn-default"><?php
             'a.entry_postcode',
         ];
         $search = zen_build_keyword_where_clause($keyword_search_fields, trim($keywords), true);
+    } else if (!empty($_GET['id'])) {
+        $search = " WHERE c.customers_id = " . (int)$_GET['id']; 
```

Comment:
This added check for `GET[id]` isn't necessary. `cID` is already built-in.

Reply:
It is already linked in the table-view of orders (see magnifying glass icon next to customer name) and in the info-sidebox (the email address is actually linked to a cID=(number) lookup).

Reply:
<img width="770" alt="Screen Shot 2024-01-30 at 11 48 57 AM" src="https://github.com/zencart/zencart/assets/404472/600cc7f5-22d0-4798-acef-ab49ae6905a1">
<img width="332" alt="Screen Shot 2024-01-30 at 11 49 15 AM" src="https://github.com/zencart/zencart/assets/404472/3619dc20-7d8f-4925-a183-5287e4231d0a">

Reply:
Right but not on the order details page where I have it now.   Will make the other change above.

Reply:
I see your point.  Fixed.